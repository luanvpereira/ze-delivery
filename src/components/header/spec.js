import React from 'react';
import { mount } from 'enzyme';

import Header from './';

describe('<Header/>', () => {
	it('should render a <Header/>', () => {
		const component = mount(<Header />);
		expect(component).toMatchSnapshot();
	});
});
