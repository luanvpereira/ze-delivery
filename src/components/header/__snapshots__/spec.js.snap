// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header/> should render a <Header/> 1`] = `
<Header>
  <header
    className="header"
  >
    <Container>
      <div
        className="container"
      >
        <h1
          className="headerTitle"
        >
          <Link
            href="/"
          >
            <a
              className="headerLogo"
              href="/"
              onClick={[Function]}
            >
              Ze Delivery
            </a>
          </Link>
        </h1>
         
      </div>
    </Container>
  </header>
</Header>
`;
